framework/generated/khronos_generators/base_generators/base_struct_handle_mappers_body_generator.py

```diff
@@ -29,60 +29,33 @@ class BaseStructHandleMappersBodyGenerator():
     """Base class for generating struct handle mappers body code."""
 
     def endFile(self):
-        platform_type = self.get_api_prefix()
-
-        if platform_type == 'Vulkan':
-            # Generate the pNext handle mapping code.
-            self.newline()
-            write(
-                'void MapPNextStructHandles(const void* value, void* wrapper, const CommonObjectInfoTable& object_info_table)',
-                file=self.outFile
-            )
-            write('{', file=self.outFile)
-            write(
-                '    if ((value != nullptr) && (wrapper != nullptr))',
-                file=self.outFile
-            )
-            write('    {', file=self.outFile)
-            write(
-                '        const {struct}* base = reinterpret_cast<const {struct}*>(value);'.format(struct=self.getBaseInputStructureName()),
-                file=self.outFile
-            )
-            write('', file=self.outFile)
-            write('        switch (base->sType)', file=self.outFile)
-            write('        {', file=self.outFile)
-            write('        default:', file=self.outFile)
-            write(
-                '            // TODO: Report or raise fatal error for unrecongized sType?',
-                file=self.outFile
-            )
-        else:
```

Comment:
Easiest to review this section with whitespace disabled, even though it's python, the removal of the `if else` for API type you can see what *hasn't* changed.